src/test/ui/underscore-lifetime/where-clauses.rs

```diff
@@ -0,0 +1,3 @@
+trait Foo<'a> {}
+impl<'b: '_> Foo<'b> for i32 {}
```

Comment:
Since the name of this test is `where-clauses.rs` I think you may have meant to use a where clause here :)